injection/config.go

```diff
@@ -46,9 +50,27 @@ func ParseAndGetRESTConfigOrDie() *rest.Config {
 		log.Fatal("Error building kubeconfig: ", err)
 	}
 
+	validBurstOrDie(*burst)
+	validQPSOrDie(*qps)
+
+	cfg.Burst = *burst
+	cfg.QPS = float32(*qps)
+
 	return cfg
 }
 
+func validBurstOrDie(burst int) {
+	if burst < 0 {
+		log.Fatal("Invalid burst value", burst)
+	}
+}
+
+func validQPSOrDie(qps float64) {
+	if qps < 0 || qps > math.MaxFloat32 {
+		log.Fatal("Invalid QPS value", qps)
+	}
+}
+
```

Comment:
Maybe inline those? Seems short enough to not make the extra mental jump.

Reply:
Done https://github.com/knative/pkg/pull/1778/commits/8c36cb88c209d63fc48e84bd1e93f6fa3d1213e2